recipes/highway/all/conandata.yml

```diff
@@ -20,21 +23,6 @@ sources:
   "0.17.0":
     url: "https://github.com/google/highway/archive/0.17.0.tar.gz"
     sha256: "25158fd5c090b70ecea47fc246c860d150f07f801d2434e1e51ec14a6c15822c"
-  "0.16.0":
-    url: "https://github.com/google/highway/archive/0.16.0.tar.gz"
-    sha256: "746c9578446be6c5286e8846c5f0d4118c0c1f04219c401abadcb8a5f2051893"
   "0.12.2":
```

Comment:
Used by libjxk - hopefully next time this gets a new version we remember to remove it once https://github.com/conan-io/conan-center-index/pull/18812 is merged :)